src/main/java/io/github/moulberry/notenoughupdates/dungeons/GuiDungeonMapEditor.java

```diff
@@ -284,7 +284,7 @@ public void drawScreen(int mouseX, int mouseY, float partialTicks) {
 							break;
 						case 8:
 							mapDesc = "Simple gray border";
-							mapCredit = "TomEngMaster";
+							mapCredit = "Madeline";
```

Comment:
Shouldnt it be [Madeleaan](https://namemc.com/profile/Madeleaan.1)

Reply:
ive just copied her discord name